must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -133,6 +133,14 @@ for ns in $namespaces; do
                 pids_ceph+=($!)
                 { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log 2>&1 &
                 pids_ceph+=($!)
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --format=json" >> "${JSON_COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log 2>&1 &
```

Comment:
Note:- Not on all images mirroring will be enabled

Reply:
So you're saying I've to check first IF mirroring is enabled on the image first?.

Reply:
can you please test this on a cluster you have 2 pools and few images in it? 1 pool without mirroring enabled and another one with mirror enabled?